analyzers/its/sources/RazorSample/RazorSample/RazorSample.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
You can delete everything but:

- _Layout
- _ValidationScriptsPartial
- _ViewImports
- _ViewStart
- Error
- Index
- Privacy
- Program